.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,187 @@
+            --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+            -o artifact-linux.zip \
+            ${{ steps.artifact-url-linux.outputs.result }}
+          mkdir artifact-linux
+          unzip -d artifact-linux artifact-linux.zip
+          lifecycle_path=$(ls artifact-linux/lifecycle-*linux.x86-64.tgz)
+          echo "::set-env name=ARTIFACT_LINUX_PATH::$PWD/$lifecycle_path"
+      - name: Download windows artifact
+        run: |
+          curl -sL -w 'RESP_CODE:%{response_code}\n' \
+            --header 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+            -o artifact-windows.zip \
+            ${{ steps.artifact-url-windows.outputs.result }}
+          mkdir artifact-windows
+          unzip -d artifact-windows artifact-windows.zip
+          lifecycle_path=$(ls artifact-windows/lifecycle-*windows.x86-64.tgz)
+          echo "::set-env name=ARTIFACT_WINDOWS_PATH::$PWD/$lifecycle_path"
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ env.LIFECYCLE_VERSION }}
+          release_name: lifecycle v${{ env.LIFECYCLE_VERSION }}
+          draft: true
+          prerelease: false
+          body: |
+            # lifecycle v${{ env.LIFECYCLE_VERSION }}
+
+            Welcome to `v${{ env.LIFECYCLE_VERSION }}`, a **beta** release of the Cloud Native Buildpack Lifecycle.
+
+            ##  Prerequisites
+
+            The lifecycle runs as a normal user in a series of unprivileged containers. To export images and cache image layers, it requires access to a Docker daemon **or** Docker registry.
+
+            ## Install
+
+            Extract the `.tgz` file and copy the lifecycle binaries into a [build stack base image](https://github.com/buildpack/spec/blob/master/platform.md#stacks). The build image can then be orchestrated by a platform implementation such as the [pack CLI](https://github.com/buildpack/pack) or [tekton](https://github.com/tektoncd/catalog/blob/master/task/buildpacks/0.1/README.md).
+      - name: Upload Release Asset - linux
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ${{ env.ARTIFACT_LINUX_PATH }}
+          asset_name: lifecycle-v${{ env.LIFECYCLE_VERSION }}+linux.x86-64.tgz
+          asset_content_type: application/gzip
+      - name: Upload Release Asset - windows
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ${{ env.ARTIFACT_WINDOWS_PATH }}
+          asset_name: lifecycle-v${{ env.LIFECYCLE_VERSION }}+windows.x86-64.tgz
+          asset_content_type: application/gzip
```

Comment:
In https://github.com/buildpacks/pack/pull/807, we actually moved from using `actions/create-release` and `actions-upload-release-assets` to using the https://github.com/softprops/action-gh-release action, which allows you to create a release and upload assets at the same time. It may not be worth moving from the defaults, given that you only upload 2 releases, but you can look into it and see.

Reply:
Ah, thanks for the tip! It's good to know that it exists. We want to hold off on changing it for now, but perhaps we'll revisit it later.